arangod/Network/Methods.cpp

```diff
@@ -111,6 +111,32 @@ auto prepareRequest(RestVerb type, std::string path, VPackBufferUInt8 payload,
   return req;
 }
 
+/// @brief Function to produce a response object from thin air:
+static std::unique_ptr<fuerte::Response> buildResponse(fuerte::StatusCode statusCode, Result const& res) {
+  VPackBuffer<uint8_t> buffer;
+  VPackBuilder builder(buffer);
+  {
+    VPackObjectBuilder guard(&builder);
+    auto errorNum = res.errorNumber();
+    builder.add(StaticStrings::Error, VPackValue(errorNum != TRI_ERROR_NO_ERROR));
+    builder.add(StaticStrings::ErrorNum, VPackValue(errorNum));
+    if (errorNum != TRI_ERROR_NO_ERROR) {
+      if (res.errorMessage().empty()) {
```

Comment:
Good point, the `Result::errorMessage()` method never returns an empty result, but creates the error message from the code if it is empty. So this can be simplified, will do.